Form/Type/FormatterType.php

```diff
@@ -41,17 +42,24 @@ class FormatterType extends AbstractType
     protected $configManager;
 
     /**
+     * @var PluginManagerInterface
+     */
+    protected $pluginManager;
+
+    /**
      * Constructor.
      *
      * @param Pool                   $pool          A Formatter Pool service
      * @param TranslatorInterface    $translator    A Symfony Translator service
      * @param ConfigManagerInterface $configManager An Ivory CKEditor bundle configuration manager
+     * @param PluginManagerInterface $pluginManager An Ivory CKEditor bundle plugin manager
      */
-    public function __construct(Pool $pool, TranslatorInterface $translator, ConfigManagerInterface $configManager)
+    public function __construct(Pool $pool, TranslatorInterface $translator, ConfigManagerInterface $configManager, PluginManagerInterface $pluginManager)
```

Comment:
Would you rather have me create and call setPluginManager() method or create PR into master?